### PR TITLE
[DEV-3064] remove duplicate bankAccount

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/supporting/bank/bank-account/bank-account.service.ts
Comment (davidleomay) [inline src/subdomains/supporting/bank/bank-account/bank-account.service.ts:1]: This can cause errors because bank accounts are linked in sells/buys

Comment (Yannick1712) [reply src/subdomains/supporting/bank/bank-account/bank-account.service.ts:1]: Fixed with DEV-3373 👍

**Comment by Yannick1712** [comment]
- [ ] Null synced col in bankAccount table

File: src/subdomains/supporting/bank/bank-account/bank-account.service.ts [hunk 1]
```diff
@@ -3,21 +3,52 @@ import { Cron, CronExpression } from '@nestjs/schedule';
 import { IbanDetailsDto, IbanService } from 'src/integration/bank/services/iban.service';
 import { CountryService } from 'src/shared/models/country/country.service';
 import { IEntity } from 'src/shared/models/entity';
+import { DfxLogger } from 'src/shared/services/dfx-logger';
 import { DisabledProcess, Process } from 'src/shared/services/process.service';
 import { Lock } from 'src/shared/utils/lock';
 import { KycType, UserData } from 'src/subdomains/generic/user/models/user-data/user-data.entity';
-import { IsNull } from 'typeorm';
+import { UserDataService } from 'src/subdomains/generic/user/models/user-data/user-data.service';
+import { IsNull, MoreThan } from 'typeorm';
```
Comment (davidleomay) [inline src/subdomains/supporting/bank/bank-account/bank-account.service.ts:11]: Unused import

**Review by davidleomay — APPROVED** [review]

**Pull request merged**